components/PresidentialList/index.js

```diff
@@ -47,6 +47,22 @@ export default function presidentList(props) {
   }
   const presidentList = data?.data;
 
+  const NewPlan = () => {
+    if (presidentList.president.org_politica_alias === 'Perú Libre')
+      return (
+        <Styled.ExternalLink
+          href={'https://perulibre.pe/plan-bicentenario.pdf'}
+          target="_blank"
+          rel="noopener noreferrer">
+          <ExternalLinkIcon />
+          <Styled.ExternalLinkText>
+            Ir al <strong>nuevo</strong> plan de gobierno
+          </Styled.ExternalLinkText>
+        </Styled.ExternalLink>
+      );
```

Comment:
```suggestion
    if (presidentList.president.org_politica_alias === 'Perú Libre') {
      return (
        <Styled.ExternalLink
          href={'https://perulibre.pe/plan-bicentenario.pdf'}
          target="_blank"
          rel="noopener noreferrer">
          <ExternalLinkIcon />
          <Styled.ExternalLinkText>
            Ir al <strong>nuevo</strong> plan de gobierno
          </Styled.ExternalLinkText>
        </Styled.ExternalLink>
      );
    }  
```

La falta de estos caracteres luego generan inconsistencies difíciles de leer, deberíamos considerar una regla de eslint https://eslint.org/docs/rules/curly